### DISCUSSION TITLE
How to change the port running after build？

### DISCUSSION THREAD

**Discussion by XiaoNode** [nuxt/nuxt#16115] [Questions]
When I run `nuxt preview` after using the `nuxt build` command, the default running port is 3000. How can I overwrite this port?

**Comment by TheAlexLichter** [comment]
Pass an env variable, e.g. `PORT=3333 nuxt preview`

**Reply by XiaoNode** [reply]
the command line is like `nuxt preview --port=3333`? When I execute it, it does not work. I can only access port 3000。

The document does not seem to explain this usage:
https://nuxt.com/docs/api/commands/preview

**Reply by TheAlexLichter** [reply]
Can you try via env variable as shown above?

**Reply by XiaoNode** [reply]
You mean through the. env file？

**Reply by TheAlexLichter** [reply]
By passing the environment variable via terminal as shown above. .env won't be read automatically after build.
